WireAuthentication/Sources/WireAuthentication/WireAuthenticationAssembly.swift

```diff
@@ -27,6 +27,8 @@ internal import WireAuthenticationLogic
 
 public struct WireAuthenticationAssembly {
 
+    private let fallbackURLScheme = "wire-sso"
```

Comment:
thought: we should inject a non-optional scheme and provide the default from the outside. What do you think?

Reply:
changed to non-optional scheme